handbook/engineering/web/goals.md

```diff
@@ -4,29 +4,44 @@
-   This includes writing more extensions ourselves, but also extending the extension API with more capabilities to enable more use cases.
-   We have a long list of ideas and use cases customers shared with us that we can solve and/or enable customers to solve with extensions.
-   Some of these integrate Sourcegraph with more external services to suit the setups of more customers, others provide completely new value no other product provides.
+_To reach our long-term goal, we set the following medium-term goals to guide our short-term iteration plans. These medium-term goals are listed in order of rought priority. This means we preference the sooner-listed goal when making progress on one would conflict with another goal, but it doesn't mean we only work on the first goal until it's done – we balance our iterations in sum so we can make some progress on all of these goals over a quarter. Within each goal, the sub-goals are also listed in (stricter) priority order._ 
+
+1. **Make the Sourcegraph web interfaces more consistent and improve discoverability of Sourcegraph features.**
+   1. Make progress on design debt throughought the web app where the web team owns the design: repo page areas and panels, navigation and headers, general theming, and the blog page (file viewer). 
+   1. Build and update designs to help users discover or understand everything Sourcegraph offers, like new navigation menus and header concepts. 
+
+1. **Make Sourcegraph extensions a core part of Sourcegraph users' experiences.**
+   1. Track anonymized, general usage of extensions to determine which extensions are most successful at adding value for our users to inform our future work.
+   1. Build new API endpoints that are robust and obviously useful (used immediately in an extension), to grow adoption of extensions. 
+   1. Make the extensions action bar clear and scalable, and help users discover and use our extensions. 
+   1. Improve discoverability and design of the extension registry to enable users to find the most useful extensions and make users excited to build their own extension. 
+
+1. **Make code insights an entirely new reason to use Sourcegraph.**
+   1. Focus on building prototypes to solve the most pressing needs of our customers, to clearly demonstrate the value of code insights. 
+   1. Expose more generalizable metrics that let our users measure and track their own goals, whether those are migrations, code smells, security needs, cross-collaboration, or other information about code. 
+   1. Expose features that let our users build their own insights. 
+
+1. **Make code host integrations reach more users more often.**
+   1. Maintain the existing native integrations and browser extensions. 
+   1. Build support for new code hosts and new browsers, like Safari and Gerrit. 
+   1. Build support for the browser extension to reference both a private Sourcegraph instance and public code on Sourcegraph.com. 
```

Comment:
What does "more often" mean in this context? Making code host integrations more reliable, increasing user engagement, or both?

Reply:
I think of "reaching more users" as "total users who've tried an integration" and doing that "more often" is basically my way of saying "they should continue to use the integration, rather than try it (still they are reached) and then stop using it." Basically a different prose-ier way of saying "increase WAU for code host integrations" but given that it's not clear, I'll update that.